sqlmesh/core/model/meta.py

```diff
@@ -62,7 +62,9 @@ class ModelMeta(_Node):
     default_catalog: t.Optional[str] = None
     depends_on_: t.Optional[t.Set[str]] = Field(default=None, alias="depends_on")
     columns_to_types_: t.Optional[t.Dict[str, exp.DataType]] = Field(default=None, alias="columns")
-    column_descriptions_: t.Optional[t.Dict[str, str]] = None
+    column_descriptions_: t.Optional[t.Dict[str, str]] = Field(
+        default=None, alias="column_descriptions"
```

Comment:
How did this work before? Couldn't use specify `column_descriptions` in a SQL model?

Reply:
We automatically infer them from inline comments in SQL models - I guess you technically could have passed a `column_descriptions_` key.

I described it as a "python model only" key in the docs, but this will allow it to be passed in SQL models too. Hmm...